src/fiatconnect/FiatDetailsScreen.tsx

```diff
@@ -139,12 +139,14 @@ const FiatDetailsScreen = ({ route, navigation }: Props) => {
         body[formFields[i].name] = inputRefs.current[i]
       }
 
+      const implicitBody: Record<string, string> = {}
```

Comment:
The values for these are not necessarily strings, since it takes whatever `param.value` is from the schema. Thats one disadvantage to doing a forEach loop here is that you have to explicitly type the variable